aws/resource_aws_athena_database_test.go

```diff
@@ -81,7 +117,7 @@ func testAccCheckAWSAthenaDatabaseDestroy(s *terraform.State) error {
 		}
 
 		rInt := acctest.RandInt()
-		bucketName := fmt.Sprintf("tf-athena-db-%s-%d", rs.Primary.Attributes["name"], rInt)
```

Comment:
In the case where I had an `_` in the db name, creating a bucket with that name in it caused an error, the random integer should be sufficient in ensuring a unique bucketName